encoding/ccf/decode.go

```diff
@@ -32,6 +32,41 @@ import (
 	cadenceErrors "github.com/onflow/cadence/runtime/errors"
 )
 
+const (
+	// Top level CCF messages are CBOR tag (tag number: CBORTagTypeDefAndValue | CBORTagTypeAndValue, content: array of 2 elements).
+	// Minimum size is 5 bytes: top level tag number (2 bytes) + min size for array of 2 elements (3 bytes).
+	minTopLevelMsgSize = 5
+)
```

Comment:
Given that it is not exported and only used in `HasMsgPrefix`, maybe move it into the function